src/alert.js

```diff
@@ -73,18 +72,10 @@ export default class extends Controller {
   }
 
   get removeAfter() {
```

Comment:
I think we can completely get rid of this method now and just use `this.removeDelayValue` directly in the other methods.

Let's do that and simplify this even more. 👍

Reply:
see https://github.com/excid3/tailwindcss-stimulus-components/pull/137/commits/86def9b6b02873fb0031d2b513bbe85165dfbd64